fastlane/Fastfile

```diff
@@ -14,6 +14,7 @@ INTERNAL_SCHEME = 'Simplenote'
 APP_STORE_SCHEME = 'Simplenote-AppStore'
 BUILD_FOLDER = 'build'
 APP_STORE_BUNDLE_IDENTIFIER = 'com.automattic.SimplenoteMac'
+APP_STORE_BUNDLE_IDENTIFIER_INTENTS = "#{APP_STORE_BUNDLE_IDENTIFIER}.Intents".freeze
```

Comment:
Because there are only two bundle identifiers for SNMac, I decided to just keep it nice and simple instead of our more complicated set ups for the other apps with more identifiers.